### PR TITLE
Feature/one day trip is not absence

### PR DESCRIPTION
**Pull request by Br3nda**
Handles when you leave today and return tomorrow -- you're present for part of each day so they count

Handles when you leave and then return on the same day -- you're still present

Does not handle departing, arriving, departing again in same day - it incorrectly decides you're present. I'll fix that existing bug in next PR

### CODE REVIEW

File: app/services/eligibility_service.rb [hunk 1]
```diff
@@ -79,27 +79,23 @@ def presence_values
 
     @client.movements.order(:carrier_date_time).each do |movement|
       if movement.direction == 'arrival'
-        presence[movement.carrier_date_time.to_date.strftime(date_format)] = true
+        presence[movement.day] = true
```
Comment (danieljauch) [inline app/services/eligibility_service.rb]: ✨

File: app/services/eligibility_service.rb [hunk 2]
```diff
@@ -79,27 +79,23 @@ def presence_values
...
-    '%Y-%m-%d'
-  end
-
   def years_before(num_years)
     (@day.to_date - num_years.year).strftime(date_format)
   end
 
+  def date_format
+    '%Y-%m-%d'
+  end
```
Comment (danieljauch) [inline app/services/eligibility_service.rb]: Should this be a helper since it's in two files?

Comment (danieljauch) [reply app/services/eligibility_service.rb]: Non-blocking comment ^

Comment (Br3nda) [reply app/services/eligibility_service.rb]: Yeah, i think maybe in config. It's the Open Fisca format.

**Pull request merged**